src/pages/cplusplus/terms-to-know-for-beginners/index.md

```diff
@@ -63,7 +63,7 @@ The output will be something different this time:
 Hello World!	 I love freeCodeCamp!
 ```
 
- If you realised , the `\t` command created a _tab space_ between the two texts . This is one kind of special command in C++. These special commands are known as *Escape Sequences* .
+ In case you didn't realize it, the `\t` command created a _tab space_ between the two blocks of text. This is one kind of special command in C++. These special commands are known as *Escape Sequences*.

```

Comment:
~~In case you didn't realize it~~ In case you didn't realize

Reply:
Your changes are good.